app/react/Metadata/helpers/formater.js

```diff
@@ -347,7 +347,7 @@ export default {
     });
   },
 
-  prepareMetadata(_doc, templates, thesauris, relationships, options = {}) {
+  prepareMetadata(_doc, templates, thesauris, relationships, options = { sortedProperties: [] }) {
```

Comment:
I think this a bit different than the previous behavior. Now, what if I pass an options object, but it doesn't have sortedProperties? The previous solution supplies that to any options object that's missing it, this one only if the entire options object is missing.

Reply:
I have changed this